html/changelogs/Sniblet-PR-15689.yml

```
@@ -0,0 +1,7 @@
+author: Sniblet
+
+delete-after: True
+
+changes:
+  - tweak: "Zeng-Hu Emotion Manipulators now give you SPAN_GOOD instead of SPAN_NOTICE messages. This means they are green."
+  - spellcheck: "Added punctuation to "feeling" artifact effects."
```

```suggestion
  - spellcheck: "Added punctuation to 'feeling' artifact effects."
```

don't use double inverted commas it messes with the formatting